src/utils.py

```diff
@@ -0,0 +1,16 @@
+def subset_hvg(adata, hvg_col):
+    """Subset highly variable genes from AnnData object
+    
+    Parameters
+    ----------
+    adata : AnnData
+        Annotated data object
+    hvg_col : str
+        Name of the boolean column in `adata.var` that contains the information if genes should be used or not
+
+    Returns
+    -------
+    AnnData
+        Copy of `adata` with subsetted genes
+    """
+    return adata[:, adata.var[hvg_col]].copy()
```

Comment:
* Rename subset_hvg to subset_vars

Either:
* Move src/utils.py to src/integrate/scvi/helper.py
* Move src/utils.py to src/utils/subset_vars.py